### ISSUE TITLE
Unable to initialize vulkan on new client

### ISSUE THREAD

**Issue by x4dr** [ValveSoftware/steam-for-linux#9677]
#### Your system information

Version: 19.06.23
Distribution: Arch rolling release
Opt-in: No
Updates: Yes, pacman -Syu and restart
* GPU: AMD (6900XT)

#### Please describe your issue in as much detail as possible:
Tried to open steam
expected: steam opens
actual: steam throws errors and dies
<script src="https://gist.github.com/x4dr/92812e44c4b5849edb30336511478686.js"></script>

#### Steps for reproducing this issue:

1. Update to new steam client
2. open steam (via terminal)

#### Logs
[steam-logs.tar.gz](https://github.com/ValveSoftware/steam-for-linux/files/11790368/steam-logs.tar.gz)

**Comment by kisak-valve** [comment]
Hello @x4dr, `vkEnumeratePhysicalDevices failed, unable to init and enumerate GPUs with Vulkan.
BInit - Unable to initialize Vulkan!` in your log is a red herring. It's likely there's an issue with a Vulkan render path on your system, but that's non-fatal to the Steam client.

The lines after that hint that you're seeing an issue simular to #9588. Can you give that issue report a read and check if you're seeing the same issue?

**Comment by vincentrinot** [comment]
I am having similar issues on ubuntu after the update.

First time i opened steam to launch Dota it gave me vulkan error... Since then ive gotten it to load some of the shaders but 9x/10 when i open steam it stops responding. Since the latest steam client.

**Comment by vincentrinot** [comment]
launching with steam -vgui seems to work for launching steam and getting past the steam errors and dying.... but then theres still the problem with Vulkan. 

It is definitely to do with the new steam patch

**Comment by x4dr** [comment]
yup, the fixed i saw in other issues do not seem to help, by now i have a helper script that just keeps calling steam in a loop until it opens, which can take 2-3 minutes but it usually has opened after a while
